index.html

```diff
@@ -116,17 +116,23 @@
             <div class="col-md-2" id="right-graveyard">
                 three
             </div>
+            </div>
+<div class="row" id="buttons-row">
+    <div class= "d-flex justify-content-center">
+        <div class="col">
+          </div>
+          <div class="col-xs-12  col-sm-12 col-md-6 col-lg-6">
+            <div class="d-flex justify-content-center">
+            <button id='new-game' type="button" class="btn btn-primary mx-4 my-2">New Game</button>
+            <button id='undo-move' type="button" class="btn btn-primary mx-4 my-2">Undo Move</button>
+            </div>
+            </div>
+            <div class="col">
+            </div>
         </div>
```

Comment:
Please fix the indentation for lines 120 - 133 so that each opening tag's indentation level matches its corresponding closing tag.